openlibrary/templates/search/authors.html

```diff
@@ -1,6 +1,8 @@
 $def with (get_results)
 
 $ q = query_param('q', '')
+$if q:
+    $ q = q.strip()
```

Comment:
Since we have a default here of an empty string, we can do this in one line. Let's do the same in both spots.
```suggestion
$ q = query_param('q', '').strip()
```

Reply:
Hm, the point of the second argument there in `query_param('q', '')` is to set the default if no query parameter is passed in. We might need to update `/search/subjects` or `/search/authors` to have this second argument, and then we can bring it down to one line!

Reply:
Updated ```search/subjects``` to have default parameter which was actually missing, which brings it down to one line.